packages/asset-service/src/generateAssetData/baseAssets.ts

```diff
@@ -37,7 +37,7 @@ export const atom: Asset = {
   icon: 'https://assets.coincap.io/assets/icons/256/atom.png',
   explorer: 'https://www.mintscan.io/cosmos',
   explorerAddressLink: 'https://www.mintscan.io/cosmos/account/',
-  explorerTxLink: 'https://www.mintscan.io/cosmos/txs/'
+  explorerTxLink: 'https://www.mintscan.io/osmosis/txs/'
```

Comment:
why are we changing this on the atom asset? i think this and the generated asset data should be reverted?

Reply:
So yea, this breaks tx's on cosmos sends/receives, but we need osmosis tx's for swaps.  Gotta figure out another solution for this.